packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -1987,13 +2007,20 @@ class _OverlayPortalState extends State<OverlayPortal> {
   @override
   Widget build(BuildContext context) {
     final int? zOrderIndex = _zOrderIndex;
+    Widget overlayChild = Builder(builder: widget.overlayChildBuilder);
+    Widget? overlayParent = widget.child;
+    if (!widget.skipSemantics) {
```

Comment:
There are several use cases in framework and customer testing uses `Positioned` in `overlayChildBuilder`, so adding a `Semantics(identifier: ...)` between the `StackParentData` and `Positioned` throws errors because Positioned requires a Stack as its direct parent. So I added this `skipSemantics` to avoid adding Semantics in between. But still feel this property is odd. Are there any other ways to either: add Semantics with an identifier for overlay portal children but not in between Stack and Positioned, or: somehow skip the Semantics and get the parentData directly from Stack?